### ISSUE TITLE
ctx.throw(Boom.badRequest(error))

### ISSUE THREAD

**Issue by erwan690** [ladjs/koa-better-error-handler#10]
`ctx.throw(Boom.badRequest(error))`

returned 

Content-Type | text/plain

how to change to 

Content-Type | application/json

**Comment by niftylettuce** [comment]
Set ctx.api to true? See README

**Comment by erwan690** [comment]
i do but it still return text 
Content-Type | text/plain

this is set 
```
app = new Koa()
    app.context.onerror = errorHandler
    app.context.api = true
    app.use(cors())
      .use(bodyParser())
      .use(methodOverride())
      .use(compression())
      .use(koa404Handler)
      .use(swaggerMiddleware)
      .use(router.routes())
```

this on the route

```
router.get('/400', ctx => {
    ctx.api = true
    ctx.throw(Boom.badRequest())
  })

```
this respond is json
![image](https://user-images.githubusercontent.com/10805800/53579216-c1e57e80-3bab-11e9-99b8-671ff79d36a7.png)

but 
Content-Type is text/plain
![image](https://user-images.githubusercontent.com/10805800/53579306-e6d9f180-3bab-11e9-93dc-c8dfcc209b47.png)

**Comment by jozsi** [comment]
Same here.

**Comment by niftylettuce** [comment]
I've fixed this in v2.0.0.  Please install this version:

```sh
npm install koa-better-error-handler@latest
```

or

```sh
yarn add koa-better-error-handler@latest
```

This version has a patch in place so that if `ctx.api = true`, by default the content type returned in responses will be `application/json` instead of `text/plain`, as you had noted in your bug report.
